cmd/edit/cluster/cmd.go

```diff
@@ -473,6 +484,93 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	/*******  AuditLogForwarding *******/
```

Comment:
Unify comments style :)

Reply:
I see a couple examples of the following comment block style being used (within the same file)
![image](https://github.com/openshift/rosa/assets/118839428/0f428501-0019-4836-b952-45fb729040c0)
![image](https://github.com/openshift/rosa/assets/118839428/36cc1bd9-f616-4eec-88b9-089d3c29666e)

Is this not appropriate for this repo? Please advise :)

Reply:
I tend to prefer 

```
//  AuditLogForwarding
```

Instead of those old classic styles

```
// /*******  AuditLogForwarding *******/
```

Anyway, it's a nitpick comment, not a blocker.